documentation/src/docs/admin-guide/data-recovery-tools.md

```diff
@@ -0,0 +1,134 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Data Recovery Tools
+
+This document provides an overview of the data recovery tools in Pravega. The target audience of this
+document are administrators of Pravega clusters that may be responsible for recovering the cluster
+after a data loss/corruption issue has happened. We strongly recommend reading this guide before
+attempting any data recovery action on a real Pravega cluster, as the consequences of a non-expert
+usage of these tools may worsen the original situation.
+
+## Tier-1 Repair Tool
+
+The Tier-1 Repair Tool allows an administrator to "edit" Durable Logs storing the sequence
+of Operations that a Segment Container has processed. For more info, please [check out the PDP](https://github.com/pravega/pravega/wiki/PDP-54-(Tier-1-Repair-Tool)).
+
+### Targeted Scenarios 
+This tool works on the actual Operations stored in a Durable Log. In most cases where 
+the data loss/corruption is related to the Operations of a Durable Log, this tool can be useful. For example,
+imagine that by some reason, the Durable Log contains a duplicate Operation written (e.g., a network glitch
+has uncovered a bug in the write path). If the Segment Container attempts to recover when there is a duplicate
+Operation in the Durable Log, the recovery process will halt throwing a `DataCorruptionException` and the DurableLog will be disabled.
+This tool can help "editing" the original Durable Log to _remove_ the duplicate Operation and recover the
+Segment Container. Similarly, if an Operation that should be there is missing, we can _add_ an Operation to fill
+that gap in the original Durable Log, or even _replacing_ a malformed Operation by another one. 
+
+### Non-Targeted Scenarios 
+This tool assumes that the metadata of the Durable Log is correct and just
```

Comment:
"Just don't say 'just'". It is a word that provides absolutely no meaning.

Reply:
Fixed.